src/actions/Entities/NameNoteActions.js

```diff
@@ -80,15 +77,16 @@ const updateForm = (data, validator) => ({
   payload: { data, isValid: validator(data) },
 });
 
-const saveEditing = () => (dispatch, getState) => {
+const saveEditing = prescriptionID => (dispatch, getState) => {
   const nameNote = selectCreatingNameNote(getState()) ?? {};
   const patient = UIDatabase.get('Name', nameNote?.nameID);
   const isDirty = JSON.stringify(patient?.mostRecentPCD?.data) !== JSON.stringify(nameNote?.data);
-
   if (isDirty) {
```

Comment:
Now that we are linking vaccination events with PCD records I'm wondering if we need to save every time (when the transaction id is updated) even if the actual PCD name note wasn't updated 🤔 .

Maybe the `isDirty` check should take into account the prescriptionID?

e.g.
If no prescriptionID: `nameNote.data.transactID = '';`
Otherwise: `nameNote.data.transactID = prescriptionID;`

Then `isDirty` check with the transactID populated:
`JSON.stringify(patient?.mostRecentPCD?.data) !== JSON.stringify(nameNote?.data);`